<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>杨文袜业🙌管理系统 V3.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/socks_icon_1.png</normaloff>:/socks_icon_1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_login">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_signin">
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="toolTip">
        <string>管理系统</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>46</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>  密码  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_pass">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>45</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="man_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>杨文袜业🙌管理系统 V3.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>user_id</tabstop>
  <tabstop>user_pass</tabstop>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_signin</tabstop>
  <tabstop>btn_exit</tabstop>
 </tabstops>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
